### PR TITLE
adding log forwarding and links to Understanding cluster logging

### PR DESCRIPTION
**Pull request by mburke5678**
Addding links to configuring page for each component and added log forwarding to the understanding cluster logging page.

### CODE REVIEW

**Comment by mburke5678** [comment]
@sichvoge WDYT?

File: logging/cluster-logging.adoc [hunk 1]
```diff
@@ -50,10 +50,20 @@ include::modules/cluster-logging-about-components.adoc[leveloffset=+2]
...
 
+For information on configuring the log visualizer, see xref:../logging/config/cluster-logging-visualizer.adoc#cluster-logging-visualizer[Configuring the log visualizer].
+
 include::modules/cluster-logging-eventrouter-about.adoc[leveloffset=+2]
 
+For information on configuring the event router, see xref:../logging/cluster-logging-eventrouter.adoc#cluster-logging-eventrouter[Collecting and storing Kubernetes events].
+
+include::modules/cluster-logging-forwarding-about.adoc[leveloffset=+2]
 
+For information on configuring log forwarding, see see xref:../logging/cluster-logging-external.adoc#cluster-logging-external[Forwarding logs to third party systems].
```
Comment (bobfuru) [inline logging/cluster-logging.adoc:69]: s/see see/see/

File: modules/cluster-logging-forwarding-about.adoc [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging.adoc
+
+[id="cluster-logging-forwarding-about_{context}"]
+= About log forwarding
+
+By default, {product-title} cluster logging sends logs to the default internal Elasticsearch log store, defined in the Cluster Logging Custom Resource. If you want to forward logs to other log aggregators, you can use the log forwarding features to send logs to specific endpoints within or outside your cluster.
```
Comment (bobfuru) [inline modules/cluster-logging-forwarding-about.adoc:8]: I believe we are now using lowercase here ("Cluster Logging custom resource"), per the [note](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#object-references) that was updated in our guidelines:
> You can use "object", "resource", "custom resource", "spec", etc. as appropriate after the object reference. This is especially recommended if it helps with clarity or readability.

**Review by bobfuru — APPROVED** [review]
A few comments but otherwise LGTM!

File: modules/cluster-logging-forwarding-about.adoc (hunk 2 above)
Comment (mburke5678) [reply modules/cluster-logging-forwarding-about.adoc:8]: @bobfuru 
Agreed. I left it upper-case as it appears that way throughout the logging docs. I should do a universal change.

**Pull request merged**